spec/system/user_page/staff_info_spec.rb

```diff
@@ -17,4 +17,45 @@
       expect(user_page).to have_warning_messages_path(user)
     end
   end
+
+  context "for flagged posts" do
+    before do
+      ReviewableFlaggedPost
```

Comment:
note: will change this to use PostActionCreator instead - this is quite brittle setup-wise.